src/test/data/JsonSerializableRoomListTest/typicalRoomsInRoomList.json

```diff
@@ -1,20 +1,62 @@
 {
-  "rooms" : [ {
```

Comment:
I think you may need to change this json file back to the original one since this json file should follow the rooms set in getTypicalRoom() from the TypicalRooms class. The getTypicalRoom() function is being used by other test cases for task and patient commands which heavily depends on the order in the room list so I am not sure if changing this file will propagate errors to those test cases for tasks and patients. :/